src/ontology/fbbt-edit.obo

```diff
@@ -118583,6 +118583,139 @@ intersection_of: expresses http://flybase.org/reports/FBgn0031903 ! Wnt10
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-23T08:58:37Z
+
+[Term]
+id: FBbt:00058023
+name: larval juvenile hormone secreting cell
+def: "An endocrine cell that secretes the juvenile hormone (JH). These cells are exclusively found in the corpus allatum from stage 17 onward (Niwa et al., 2008)." [FlyBase:FBrf0205213]
+subset: cur
+intersection_of: FBbt:00058020 ! juvenile hormone secreting cell
+intersection_of: part_of FBbt:00001715 ! embryonic/larval endocrine system
+relationship: part_of FBbt:00001725 ! embryonic/larval corpus allatum
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-23T10:14:16Z
+
+[Term]
+id: FBbt:00058024
+name: adult DMS cell of pars intercerebralis
+def: "A dromyosuppressin-positive neuron of the pars intercerebralis. There are approximately 30 of these cells. They have a single process that bifurcates into a long axonal projection to the hypocerebral ganglion and the crop, and a shorter dendritic process that reaches the suboesophageal zone. They exert a relaxing activity on the crop muscle, notably in response to increasing levels of the bursicon and ecdysone hormones. They may also innervate the hindgut, the rectal ampulla, the heart, and the female reproductive tract (Hadjieconomou et al., 2020)." [FlyBase:FBrf0247320]
+subset: cur
+intersection_of: FBbt:00058028 ! myosuppressin-positive neuron
+intersection_of: has_soma_location FBbt:00003700 ! adult pars intercerebralis
+relationship: expresses http://flybase.org/reports/FBgn0000546 ! EcR
+relationship: expresses http://flybase.org/reports/FBgn0003255 ! rk
+relationship: fasciculates_with FBbt:00004050 ! crop nerve
+relationship: RO:0002360 FBbt:00110639 ! has dendrite location adult subesophageal zone
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-23T11:27:10Z
+
+[Term]
+id: FBbt:00058025
+name: adult DMS cell of hypocerebral ganglion
+def: "A dromyosuppressin-positive neuron of the hypocerebral ganglion. There are approximately 5 of these cells. They project to the crop and exert a relaxing activity on the crop muscle (Hadjieconomou et al., 2020)." [FlyBase:FBrf0247320]
+subset: cur
+intersection_of: FBbt:00058028 ! myosuppressin-positive neuron
+intersection_of: has_soma_location FBbt:00004047 ! adult hypocerebral ganglion
+relationship: fasciculates_with FBbt:00004050 ! crop nerve
+created_by: http://orcid.org/0000-0002-6095-8718
+creation_date: 2021-04-23T11:35:54Z
+
+[Term]
+id: FBbt:00058026
+name: adult mechanosensory neuron of the hypocerebral ganglion
+def: "A mechanosensory neuron expressing the mechanosensitive ion channel Piezo and innervating the anterior gut. There are approximately 6 of these neurons in the hypocerebral ganglion. They respond to crop-distending stimuli and control feeding behavior (Min et al., 2021)." [FlyBase:FBrf0248237]
+subset: cur
+intersection_of: FBbt:00005106 ! neuron
+intersection_of: expresses http://flybase.org/reports/FBgn0264953 ! Piezo
```

Comment:
I would use "'capable of' some 'detection of mechanical stimulus involved in sensory perception'" in the logical definition (this will also autoclassify this as a 'mechanosensory neuron') and move "expresses some Piezo" to a SubClassOf relationship.